sdk/storage/azure-storage-common/pom.xml

```diff
@@ -121,20 +129,19 @@
 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jar-plugin</artifactId>
-        <version>3.1.2</version> <!-- {x-version-update;org.apache.maven.plugins:maven-jar-plugin;external_dependency} -->
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>3.2.0</version>
         <executions>
           <execution>
-            <id>test-jar</id>
-            <phase>test-compile</phase>
+            <id>test-shared-jar</id>
+            <phase>process-test-classes</phase>
             <goals>
-              <goal>test-jar</goal>
+              <goal>single</goal>
             </goals>
             <configuration>
-              <includes>
-                <include>com/azure/storage/common/test/shared/**/*</include>
-                <include>SpockConfig.groovy</include>
-              </includes>
+              <descriptors>
+                <descriptor>src/test-shared/assembly/assembly.xml</descriptor>
+              </descriptors>
```

Comment:
how does the output of this look like in target folder when you run mvn install?

@alzimmermsft does this affect how commons is released to mvncentral anyhow that we should be worried about?

Reply:
maven doesnt copy over the assembly folder/file into target since it's not specified as a resource. the manifests get transferred as a `manifests` folder into the `target/test-classes` folder, preventing it from getting caught with common tests, and the assembly plugin maps it where it needs to go in the artifact

Reply:
yeah, but does test jar look the same ?

Reply:
yeah, I ran `jar xf` on it and the contents are the same.

Reply:
This shouldn't affect releasing to Maven central as this only affect building the library from source